src/main/java/uk/gov/hmcts/ccd/domain/enablingcondition/EnablingConditionFormatter.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.hmcts.ccd.domain.casestate;
+package uk.gov.hmcts.ccd.domain.enablingcondition;
 
 public interface EnablingConditionFormatter {
```

Comment:
```suggestion
public interface EnablingConditionConverter {
```
also can you add a small Javadoc explaining this is converting our internal enabling condition syntax to the Jexl one?

Reply:
and change the method:
    `String format(String enablingCondition);`
`String convertToJexl(String enablingCondition);`

Reply:
sorry the javadoc should be on the JexlConverter subclass not on the interface. On the interface we can add a more generic description of the class withouth mentioning Jexl

Reply:
done